rmw_zenoh_cpp/src/zenohd/main.cpp

```diff
@@ -192,20 +194,26 @@ int main(int argc, char ** argv)
     return 1;
   }
 
-  z_owned_session_t s = z_open(z_move(config));
-  if (!z_check(s)) {
+  z_owned_session_t session = z_open(z_move(config));
+  if (!z_check(session)) {
     printf("Unable to open router session!\n");
     return 1;
   }
   printf(
     "Started Zenoh router with id %s.\n",
-    rmw_zenoh_cpp::liveliness::zid_to_str(z_info_zid(z_session_loan(&s))).c_str());
+    rmw_zenoh_cpp::liveliness::zid_to_str(z_info_zid(z_session_loan(&session))).c_str());
 #ifdef _WIN32
   SetConsoleCtrlHandler(quit, TRUE);
 #else
   signal(SIGINT, quit);
+  signal(SIGTERM, quit);
 #endif
 
+  auto close_session = rcpputils::make_scope_exit(
```

Comment:
Sorry one tiny nitpick, can we move this `close_session` instantiation right after the `if (!z_check(session))` block? Just to be consistent with other parts of the codebase.

Reply:
Done it with this commit (https://github.com/ros2/rmw_zenoh/pull/252/commits/35c5ba0fb81878796ab75bfa705f241daf87fd1d)